.azure-pipelines/generation-templates/generate-service-modules.yml

```diff
@@ -138,7 +142,12 @@ jobs:
           script: |
             [bool]$EnableSigning = if ("$(EnableSigning)" -eq "true") { $true } else { $false }
             $Modules = "$(ModulesToGenerate)" -split " "
-            . $(System.DefaultWorkingDirectory)/tools/GenerateModules.ps1 -Build -Test -UpdateAutoRest -EnableSigning:$EnableSigning -ModulesToGenerate $Modules
+            . $(System.DefaultWorkingDirectory)/tools/GenerateModules.ps1 -Build -Test -UpdateAutoRest -EnableSigning:$EnableSigning -ModulesToGenerate $Modules -ExcludeExampleTemplates
+       
+      - template: ./generation-templates/generate-helpdocs-template.yml
+        parameters:
+            DOCSBRANCH: $(DocsBranch)
+            ModulesToGenerate: $(ModulesToGenerate)
```

Comment:
Commit https://github.com/microsoftgraph/msgraph-sdk-powershell/pull/726/commits/43a48139dd7e48c3eb1c6132f19b73ac74832bb1 removed `ModulesToGenerate` parameter from `generate-helpdocs-template.yml`, yet this line still uses it. This change will break the weekly API refresh.

Reply:
Good catch.